tests/crashes/133065.rs

```diff
@@ -0,0 +1,15 @@
+//@ known-bug: #133065
+//@compile-flags: -Zvalidate-mir -Zinline-mir -Zinline-mir-threshold=300
```

Comment:
Suggestion: hmm, this is also one of those cases where it's better to identify which mir opts are involved, like #133104. Probably split this off into another PR.

Reply:
as far as I understand this only enables mir inlining.
does mir inlining also enable other opts than mir inlining

Reply:
cc @saethlin do you know? I'm not super familiar with the MIR-specific ones.

Reply:
I'm not too convinced that this is too useful in general.
If we for example have a broken mir opt in the pipeline, and the mir opt gets banished to -Zunsave-mir-opts, this still fixes the bug from a user perspective as the mir opt is no longer run by default.

Also some mir opts rely on each other which is why I don't fuzz single mir opts, as there are probably bunch of cases where "one opt will break if that other opt is not run before it."

Reply:
Since this is a crashes test we can also remove it easily if it becomes a problem, so yeah it's a whatever.